engine/src/main/java/org/terasology/rendering/dag/WireframeStateChange.java

```diff
@@ -16,11 +16,20 @@
 package org.terasology.rendering.dag;
 
 
+import static org.lwjgl.opengl.GL11.GL_FILL;
+
 /**
  *
  */
 public class WireframeStateChange extends AbstractStateChange<Integer> {
```

Comment:
In line with my comment above, I'd rename this "SetWireframeStateChange" or "SetWireframe", to have a verb in the name.